storybook/cypress/e2e/header.cy.ts

```diff
@@ -70,104 +71,113 @@ describe("Header", () => {
       .dsoCheckA11y("dso-header.hydrated");
   });
 
-  it("should show/remove dropdownmenu", () => {
-    cy.get("dso-header")
+  it("should show and remove dropdownmenu", () => {
+    cy.get("@dsoHeaderShadow")
       .find("nav")
       .should("be.visible")
-      .get("dso-header")
+      .get("@dsoHeaderShadow")
       .find(".dropdown dso-dropdown-menu")
       .should("not.exist")
       .viewport(991, 600)
-      .get("dso-header")
+      .get("@dsoHeaderShadow")
       .find("nav")
       .should("not.exist")
-      .get("dso-header")
+      .get("@dsoHeaderShadow")
       .find(".dropdown dso-dropdown-menu")
       .should("be.visible")
       .viewport(992, 600)
-      .get("dso-header")
+      .get("@dsoHeaderShadow")
       .find("nav")
       .should("be.visible")
-      .get("dso-header")
+      .matchImageSnapshot(`${Cypress.currentTest.title} -- visible`)
```

Comment:
De naam van de snapshot overschrijven (of suffixen in dit geval) is alleen relevant als er meer dan één `.matchImageSnapshot()` in een test (`it()`) staat.

Waarom heb je dit gedaan?

Reply:
Ik denk dat ik eerst 2 screenshots heb willen maken.